### ISSUE TITLE
backend: create new schema which is saved on lissi backend

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#51]
## User story
1. As a {HR-employee}
2. I want / need {the DIdentity backend to be able to create a new schema via communication with the Lissi backend}
3. So that {the schema is created in Lissi}

## Acceptance criteria
* a request to the Lissi API to create new schema can be sent
* new schema is actually created in the Lissi-backend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Idontker** [comment]
@jackDS008 Ich bin mit #62 fast fertig. Allerdings muss ich mit dem PR auf dich warten, da in der acceptance criteria drinnen steht, dass der Prozess laufen soll.
Magst du mich taggen, wenn du fertig bist?
